certbot/tests/cert_manager_test.py

```diff
@@ -69,7 +72,7 @@ def _set_up_config(self, domain, custom_archive):
         return config
 
     def tearDown(self):
-        shutil.rmtree(self.tempdir)
+        super(BaseCertManagerTest, self).tearDown()
```

Comment:
I think we can just delete the definition of `tearDown` here as it's defined in the base class.